test/mock/core/host_api/host_api_mock.hpp

```diff
@@ -260,7 +260,7 @@ namespace kagome::host_api {
 
     // -------------------------- Offchain extension ---------------------------
 
-    MOCK_METHOD(runtime::WasmI8,
+    MOCK_METHOD(runtime::WasmU32,
```

Comment:
I have doubt about this change... Look at spec:
![image](https://user-images.githubusercontent.com/9251127/147483527-7e5c87b9-bd2c-4342-8eaf-e542b64c4615.png)